### PR TITLE
Add Zebra_Datepicker@1.9.5 w/ git auto-update

### LINKED ISSUE #8552: [Request] Add Zebra_DatePicker 1.8.9
**Library name:** Zebra_DatePicker
**Git repository url:** https://github.com/stefangabos/Zebra_Datepicker
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/36160907-request-add-zebra_datepicker-1-8-9?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by NiniHsu**
close #8552, cc @stefangabos

Pull request for issue: #8552
Related issue(s): # #
@kennynaoh please review this PR, thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/stefangabos/Zebra_Datepicker
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: LGPL-3.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 15
   - Count of stars: 239
   - Count of forks: 151
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by NiniHsu** [comment]
@kennynaoh I see what you mean. Please review the PR, thank you!

**Comment by kennynaoh** [comment]
I think we also need to add `css/` and `images/` directory. How do you think?

**Comment by NiniHsu** [comment]
I add the folders. @kennynaoh please review the PR, thank you!

File: ajax/libs/Zebra_Datepicker/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "iso-8601",
+    "ui"
+  ],
+  "license": "LGPL-3.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/stefangabos/Zebra_Datepicker.git",
+    "fileMap": [
+      {
+        "basePath": "public/javascript",
```
Comment (kennynaoh) [inline ajax/libs/Zebra_Datepicker/package.json]: Wrong `basePath`. Could you use auto-update to check what files what you grab? I think this `basePath` can't get the files under `css/` and `images/`.

**Comment by sufuf3** [comment]
@NiniHsu would you like to finish this PR, ASAP?

**Comment by stefangabos** [comment]
In the meantime the folder structure has changed and I think the entire process of adding Zebra_DatePicker to cdnjs has to be started over...

**Comment by sufuf3** [comment]
@stefangabos Thank you for your comment, let's wait for @NiniHsu's response.
 If there is no response in 7 days, I will add this lib ASAP.
Thank you.

**Comment by sufuf3** [comment]
@PeterDaveHello Can we close this PR?
Because there is no response from the contributor since Mar 24.
Thanks

**Comment by sufuf3** [comment]
Hello @NiniHsu thanks for contribution.
Would you please help to work on this and help improve this PR?
If you have any question, please let us know.

Many thanks.

**Comment by sashberd** [comment]
Closed due to already exists via npm. Opend #12920 to manually add old versions from git

**Pull request closed without merging**